Source/santad/DataLayer/SNTEventTable.m

```diff
@@ -98,8 +98,9 @@ - (BOOL)addStoredEvent:(SNTStoredEvent *)event {
 
   __block BOOL success = NO;
   [self inTransaction:^(FMDatabase *db, BOOL *rollback) {
-    success = [db executeUpdate:@"INSERT INTO 'events' (filesha256, eventdata) VALUES (?, ?)",
-                                event.fileSHA256, eventData];
+    success = [db executeUpdate:@"INSERT INTO 'events' (idx, filesha256, eventdata)"
+                                @"VALUES (?, ?, ?)",
+                                event.idx, event.fileSHA256, eventData];
```

Comment:
Not likely as it's only caller sets idx right before calling the method. Added idx to the nil early return check at the beginning of the method just in case.